ceph.spec.in

```diff
@@ -22,7 +22,13 @@
 # bcond syntax!
 #################################################################################
 %bcond_without ocf
+# dependencies for running "make check"
 %bcond_with make_check
+# runtime dependencies for testing ceph. please note, some of them are covered
+# by "make_check" dependencies. they are listed as BuildRequires simply because
+# install-deps.sh uses {yum,dnf} builddep for preparing the develop environment
```

Comment:
Please change `{yum,dnf} builddep` to `BuildRequires` (this is a spec file, after all)

Reply:
i am sorry for this! just wanted to be more specific. i changed it to

> # install-deps.sh installs "BuildRequires" packages for preparing the develop
> # environment of Ceph.

Reply:
qa_run removed along with this comment.